xbmc/pvr/epg/EpgInfoTag.h

```diff
@@ -266,6 +266,12 @@ namespace PVR
      */
     int GenreSubType() const;
 
+    /*!
+     * @brief Get the genre description of this event.
+     * @return The genre.
+     */
+    std::string GenreDescription() const;
```

Comment:
Maybe it makes sense to use as `const std::string&`?

Not sure what a compiler does on the e.g. `tag.GenreDescription().c_str()`, but maybe still a `malloc` about copied string used :thinking:.

Reply:
Good point. Will be changed.

Reply:
how about returning `std::string_view` and change `tag.GenreDescription().c_str()` to `...Description().data()` ?
though this looks more like something for a refactor

Reply:
> though this looks more like something for a refactor

I will not touch this in this PR.